Components/typography/typography.html

```diff
@@ -0,0 +1,230 @@
+                            <iframe
+                                src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C0%29&t=paraiso-dark&wt=none&l=css&ds=true&dsyoff=0px&dsblur=0px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Droid+Sans+Mono&fs=14px&lh=209%25&si=false&es=2x&wm=false&code=%253Cp%2520class%253D%2522para-lg%2522%253EP-Paragraph-large-36px%252F3.6rem%253C%252Fp%253E%250A%253Cp%2520class%253D%2522para-md%2522%253EP-Paragraph-medium-24px%252F2.4rem%253C%252Fp%253E%250A%253Cp%2520class%253D%2522para-rg%2522%253EP-Paragraph-regular-21px%252F2.1rem%253C%252Fp%253E%250A%253Cp%2520class%253D%2522para-sm%2522%253EP-Paragraph-small-18px%252F1.8rem%253C%252Fp%253E"
+                                style="width: 1024px; height: 173px; border:0; transform: scale(1); overflow:hidden;"
+                                sandbox="allow-scripts allow-same-origin">
+                            </iframe>
+                        </div>
+
+
+                        <div class="examples__heading-tag">
+                            <h2>Example of Aligned Paragraph text</h2>
+                            <p>To position the text, apply the <span class="blue-text">text-center, text-left, and
+                                    text-right</span> classes to the wrapper element.
+
+                        </div>
+
+                        <div class="examples-box-align">
+                            <div class="paragraph-align">
+                                <h3 class="text-left">Heading-left <span class="blue-text">1</span></h3>
+                                <h3 class="text-center">Heading-center <span class="blue-text">2</span></h3>
+                                <h3 class="text-right">Heading-right <span class="blue-text">3</span></h3>
+                            </div>
+                        </div>
+
+                        <div class="iframe">
+                            <iframe
+                                src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C0%29&t=paraiso-dark&wt=none&l=css&width=680&ds=true&dsyoff=0px&dsblur=0px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Droid+Sans+Mono&fs=14px&lh=209%25&si=false&es=2x&wm=false&code=%253Ch3%2520class%253D%2522text-left%2522%253EHeading-left%2520%253Cspan%2520class%253D%2522blue-text%2522%253E1%253C%252Fspan%253E%253C%252Fh3%253E%250A%253Ch3%2520class%253D%2522text-center%2522%253EHeading-center%2520%253Cspan%2520class%253D%2522blue-text%2522%253E2%253C%252Fspan%253E%253C%252Fh3%253E%250A%253Ch3%2520class%253D%2522text-right%2522%253EHeading-right%2520%253Cspan%2520class%253D%2522blue-text%2522%253E3%253C%252Fspan%253E%253C%252Fh3%253E"
+                                style="width: 680px; height: 145px; border:0; transform: scale(1); overflow:hidden;"
+                                sandbox="allow-scripts allow-same-origin">
+                            </iframe>
+                        </div>
+                    </div>
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
you can remove this blank line & reduce line of code in file

Reply:
done